ssv/spectest/tests/runner/postconsensus/unknown_signer.go

```diff
@@ -30,7 +30,8 @@ func UnknownSigner() tests.SpecTest {
 				Messages: []*types.SSVMessage{
 					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PostConsensusSigSyncCommitteeContributionWrongSignerMsg(ks.Shares[1], 5, 5, ks)),
```

Comment:
I actually don't get much the difference between this test and `inconsistent_beacon_signer`... Maybe the latter shouldn't be using `PostConsensusSigSyncCommitteeContributionWrongSignerMsg`?

Reply:
opened  PR #272

Reply:
inconsistent tests same msg with inconsistent signers and unknown signer tests an unknown signer (not part of the committee)